dev/README.md

```diff
@@ -31,3 +34,38 @@ The script currently has the following limitations:
 - It only works with Excel spreadsheets as input.
 - It does not generate default values for rules. Default values must be added manually if they are not present in the input spreadsheet.
 - Rules rego implementation is required before running the script. The script will fail if the rego implementation is not present.
+
+### Generate Rule Templates
+
+`generate_rule_templates.py` generate the rule templates that will show in our Kibana plug-in (csp-rules).
+
+**Usage:**
+
+From the root dir you can run the following example to generate selected benchmark rules templates
+
+```shell
+poetry run python dev/generate_rule_templates.py --benchmark <benchmark_id> --rules <selected rules>
+```
+
+**Example 1** - Generate all rules templates from all benchmarks:
+
+```shell
+poetry run python dev/generate_rule_templates.py
```

Comment:
This command returns an error:
```
Traceback (most recent call last):
  File "/Users/ofirlapid/dvlp/csp-security-policies/forks/ofiriro3/csp-security-policies/dev/generate_rule_templates.py", line 115, in <module>
    generate_rule_templates(benchmark_id, args.rules, args.out)
  File "/Users/ofirlapid/dvlp/csp-security-policies/forks/ofiriro3/csp-security-policies/dev/generate_rule_templates.py", line 20, in generate_rule_templates
    if rule not in selected_rules and rule.removeprefix("cis_").replace("_", ".") not in selected_rules:
```

Reply:
As far as I understand, if I don't choose rules at all the problem reproduce

Reply:
let me double check, default value should be all

Reply:
Was fixed